extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisReceiver.scala

```diff
@@ -217,6 +226,29 @@ private[kinesis] class KinesisReceiver[T](
   }
 
   /**
+   * Set the checkpointer that will be used to checkpoint sequence numbers to DynamoDB for the
+   * given shardId.
+   */
+  private[kinesis] def setCheckpointer(
```

Comment:
The whole class is `private[kinesis]`. Its not a big deal really, but it causes a bit of confusion while reading the code, as if this the whole class is public. Please be aware of such issues elsewhere.